portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,37 +33,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  private ArrayList<String> shows;
-
+ 
   @Override
-  public void init(){
-      shows = new ArrayList<String>();
-      shows.add("The Good Doctor");
-      shows.add("Grey\'s Anatomy");
-      shows.add("House");
-      shows.add("Code Black");
-  }
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //String show = shows.get((int) (Math.random() * shows.size()));
-    //response.setContentType("text/html;");
-    //response.getWriter().println(show);
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Hey Anesia, no need to remove the doGet() function! You just have to add doPost() as a new function to the servlet. The SubtractionServlet.java file in the walkthrough contains both, as an example.

All you need to do to your doGet() function is to remove the step where you add your hardcoded messages, which you already did.